Drift/config/services.yaml

```diff
@@ -0,0 +1,19 @@
+parameters:
+    app_path: "../.."
+
+# config/services.yaml
+services:
+    _defaults:
+        autowire: true
+        autoconfigure: true
+
+    App\:
+        resource: '%kernel.project_dir%/src/*'
+        exclude: '%kernel.project_dir%/src/{DependencyInjection,Entity,Migrations,Tests}'
+    #
+    # Controllers
+    #
+    App\Controller\:
```

Comment:
Same here.

Not all DriftPHP will use Controllers.

Reply:
I just copied this one over from the existing `skeleton/Drift/config/services.yml`

Reply:
Oh, then my fault.

We could comment that part then.
Thanks, and sorry!

Reply:
;)